project-api/project-api-test/src/test/resources/myproject_with_lang_import/main.bal

```diff
@@ -0,0 +1,7 @@
+import myproject.services;
+import sameera/myproject.storage;
+
+public function main() {
+services:runServices();
+storage:initDatabase();
+}
```

Comment:
```suggestion
public function main() {
    services:runServices();
    storage:initDatabase();
}
```

Reply:
added in 95dc03b438c8c66c9f0ccf3fa841e548ff1688a9 commit